modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ResultsBar.js

```diff
@@ -22,14 +22,33 @@ const ResultsBar = ({
 	clearResultsURL,
 	filterLabelItems,
 	itemsTotal,
+	searchContainerId,
 	searchValue,
 }) => {
 	const resultsBarRef = useRef();
 
+	const searchContainerRef = useRef();
+
+	useEffect(() => {
+		Liferay.componentReady(searchContainerId).then((searchContainer) => {
+			searchContainerRef.current = searchContainer;
+		});
+
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, []);
+
 	useEffect(() => {
 		resultsBarRef.current?.focus();
 	}, [searchValue]);
 
+	const handleClick = (event) => {
+		event.preventDefault();
+
+		searchContainerRef.current?.fire('clearFilter');
+
+		window.location.href = clearResultsURL;
```

Comment:
We should use `navigate` util here, imported from `frontend-js-web`. It will use SPA if enabled.